### PR TITLE
Stop linking to proposed rules

### PR DESCRIPTION
**Pull request by WilcoFiers**
Since we have redirects set up now, if the rule isn't approved yet, we don't need to link to proposed rules anymore.

Need for Call for Review: none, editorial stuff

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Call for Review period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: _rules/bypass-blocks-cf77f2.md [hunk 1]
```diff
@@ -506,12 +506,12 @@ This [document][] is not an [HTML web page][].
 ```
 
 [block of content]: #block-of-content 'Definition of Block of Content'
-[block collapsible]: https://www.w3.org/WAI/standards-guidelines/act/rules/3e12e1/proposed/ 'Rule Block of Repeated Content is Collapsible'
+[block collapsible]: https://www.w3.org/WAI/standards-guidelines/act/rules/3e12e1// 'Rule Block of Repeated Content is Collapsible'
```
Comment (carlosapaduarte) [inline _rules/bypass-blocks-cf77f2.md]:
```suggestion
[block collapsible]: https://www.w3.org/WAI/standards-guidelines/act/rules/3e12e1/ 'Rule Block of Repeated Content is Collapsible'
```

**Review by Jym77 — APPROVED** [review]

File: _rules/bypass-blocks-cf77f2.md [hunk 2]
```diff
@@ -506,12 +506,12 @@ This [document][] is not an [HTML web page][].
...
 [block of repeated content]: #block-of-repeated-content 'Definition of Block of Repeated Content'
 [document]: https://dom.spec.whatwg.org/#concept-document 'DOM definition of Document'
-[document has landmark]: https://www.w3.org/WAI/standards-guidelines/act/rules/b40fd1/proposed/ 'Rule Document Has a Landmark with Non-Repeated Content'
-[document has instrument to main]: https://www.w3.org/WAI/standards-guidelines/act/rules/ye5d6e/proposed/ 'Rule Document Has an Instrument to Move Focus to Non-Repeated Content'
-[document has heading for main]: https://www.w3.org/WAI/standards-guidelines/act/rules/047fe0/proposed/ 'Rule Document Has Heading for Non-Repeated Content'
+[document has landmark]: https://www.w3.org/WAI/standards-guidelines/act/rules/b40fd1// 'Rule Document Has a Landmark with Non-Repeated Content'
```
Comment (carlosapaduarte) [inline _rules/bypass-blocks-cf77f2.md]:
```suggestion
[document has landmark]: https://www.w3.org/WAI/standards-guidelines/act/rules/b40fd1/ 'Rule Document Has a Landmark with Non-Repeated Content'
```

File: _rules/bypass-blocks-cf77f2.md [hunk 3]
```diff
@@ -506,12 +506,12 @@ This [document][] is not an [HTML web page][].
...
+[document has instrument to main]: https://www.w3.org/WAI/standards-guidelines/act/rules/ye5d6e// 'Rule Document Has an Instrument to Move Focus to Non-Repeated Content'
```
Comment (carlosapaduarte) [inline _rules/bypass-blocks-cf77f2.md]:
```suggestion
[document has instrument to main]: https://www.w3.org/WAI/standards-guidelines/act/rules/ye5d6e/ 'Rule Document Has an Instrument to Move Focus to Non-Repeated Content'
```

File: _rules/bypass-blocks-cf77f2.md [hunk 4]
```diff
@@ -506,12 +506,12 @@ This [document][] is not an [HTML web page][].
...
+[document has heading for main]: https://www.w3.org/WAI/standards-guidelines/act/rules/047fe0// 'Rule Document Has Heading for Non-Repeated Content'
```
Comment (carlosapaduarte) [inline _rules/bypass-blocks-cf77f2.md]:
```suggestion
[document has heading for main]: https://www.w3.org/WAI/standards-guidelines/act/rules/047fe0/ 'Rule Document Has Heading for Non-Repeated Content'
```

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]
Some trailing slashes left behind

**Review by carlosapaduarte — APPROVED** [review]

**Review by daniel-montalvo — APPROVED** [review]

**Pull request merged**